internal/acceptance/steps.go

```diff
@@ -87,7 +88,11 @@ func (td TestData) CheckWithClientForResource(check ClientCheckFunc, resourceNam
 			if err != nil {
 				return fmt.Errorf("building client: %+v", err)
 			}
-			return check(client.StopContext, client, rs.Primary)
+
+			ctx, cancel := context.WithTimeout(client.StopContext, 5*time.Minute)
+			defer cancel()
```

Comment:
FWIW we've intentionally not configured a default timeout/check here since the default timeout isn't going to be sufficient in some cases, and since we're adding a default timeout here, we'd end up hitting the 5m timeout before any downstream longer timeouts could be fully used - so unfortunately I don't think there's a reasonable way for us to have a default timeout set here?

Rather than doing this, could we instead set a default timeout on the test specific check function?

Reply:
Makes sense, will do that instead 👍